concourse/pipelines/create-deployer.yml

```diff
@@ -0,0 +1,299 @@
+      config:
+        image: docker:///governmentpaas/bosh-init
+        inputs:
+        - name: paas-cf
+        - name: vpc-terraform-state
+        outputs:
+        - name: vpc-terraform-outputs
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            ruby paas-cf/concourse/scripts/extract_tf_vars_from_terraform_state.rb \
+            < vpc-terraform-state/vpc.tfstate \
+            > vpc-terraform-outputs/tfvars.sh
+            ls -l vpc-terraform-outputs/tfvars.sh
+            cat vpc-terraform-outputs/tfvars.sh
+            ruby paas-cf/concourse/scripts/extract_terraform_state_to_yaml.rb \
+            < vpc-terraform-state/vpc.tfstate \
+            > vpc-terraform-outputs/vpc-terraform-outputs.yml
+            ls -l vpc-terraform-outputs/vpc-terraform-outputs.yml
+            cat vpc-terraform-outputs/vpc-terraform-outputs.yml
+
+    - task: terraform-apply
+      config:
+        image: docker:///governmentpaas/docker-terraform
+        inputs:
+        - name: paas-cf
+        - name: vpc-terraform-outputs
+        params:
+          DEPLOY_ENV: {{deploy_env}}
+          AWS_DEFAULT_REGION: {{aws_region}}
+          AWS_ACCESS_KEY_ID: {{aws_access_key_id}}
+          AWS_SECRET_ACCESS_KEY: {{aws_secret_access_key}}
+          TF_VAR_AWS_ACCESS_KEY_ID: {{aws_access_key_id}}
+          TF_VAR_AWS_SECRET_ACCESS_KEY: {{aws_secret_access_key}}
+          TF_VAR_concourse_db_password: {{concourse_db_password}}
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            . vpc-terraform-outputs/tfvars.sh
+            cd paas-cf/terraform/concourse
+            terraform remote config -backend=s3 -backend-config={{tfstate_bucket}} -backend-config="key=concourse.tfstate"
+            terraform apply -var env={{deploy_env}}
+
+    - task: concourse-terraform-outputs
+      config:
+        image: docker:///governmentpaas/bosh-init
+        inputs:
+        - name: paas-cf
+        - name: concourse-terraform-state
+        outputs:
+        - name: concourse-terraform-outputs
+        run:
+          path: sh
+          args:
+          - -e
+          - -c
+          - |
+            ruby paas-cf/concourse/scripts/extract_terraform_state_to_yaml.rb \
+            < concourse-terraform-state/concourse.tfstate \
+            > concourse-terraform-outputs/concourse-terraform-outputs.yml
+            ls -l concourse-terraform-outputs/concourse-terraform-outputs.yml
+            cat concourse-terraform-outputs/concourse-terraform-outputs.yml
+
+    - task: create-concourse-secrets
+      config:
+        params: {aws_access_key_id: {{aws_access_key_id}} }
+        params: {aws_secret_access_key: {{aws_secret_access_key}} }
+        params: {concourse_atc_password: {{concourse_atc_password}} }
+        params: {concourse_db_password: {{concourse_db_password}} }
+        params: {private_ssh_key: {{private_ssh_key}} }
```

Comment:
Any reason for different params declaration format? is this equivalent or does it mean st. else?